grpc-common/README.md

```diff
@@ -159,7 +173,7 @@ Read the [section below](#providing-own-interceptors) before developing your own
 ### Providing context
```

Comment:
Could you please revise this section? It's not fully clear for me.

E.g. `Only String headers are supported` - is it still valid?

Also I'm not sure what do you want to demonstrate with the code shown below `content is a case class inserted into the Context by an interceptor like this one`.

Thanks!

Reply:
That you're able to provide your handler a context class which may contain non-trivial fields (not only headers). This is used e.g. in Authenticator where `access token` is transformed to a valid authentication.

Reply:
Reworked.